internal/server/storage/pool_interface.go

```diff
@@ -143,7 +143,7 @@ type Pool interface {
 	MigrateCustomVolume(projectName string, conn io.ReadWriteCloser, args *migration.VolumeSourceArgs, op *operations.Operation) error
 
 	// Custom volume backups.
-	BackupCustomVolume(projectName string, volName string, writer instancewriter.InstanceWriter, optimized bool, snapshots bool, op *operations.Operation) error
+	BackupCustomVolume(projectName string, volName string, contentType drivers.ContentType, writer instancewriter.InstanceWriter, optimized bool, snapshots bool, op *operations.Operation) error
```

Comment:
It feels pretty weird from an API point of view having to pass in the contentType.

I understand that we technically already have it in the caller and that saves us a second lookup, but I think I'd keep this function signature the way it was to keep that (internal) API clean.

Reply:
Alright